routes.go

```diff
@@ -33,11 +33,11 @@ func setupRoutes(e *echo.Echo) {
 		r.POST("/bulk_create", BulkCreate, append(permissionMiddleware, middleware.UserCatcher)...)
 
 		// Sources
-		r.GET("/sources", SourceList, tenancyWithListMiddleware...)
-		r.GET("/sources/:id", SourceGet, middleware.Tenancy)
+		r.GET("/sources", SourceList, append(tenancyWithListMiddleware, middleware.UserCatcher)...)
```

Comment:
So rather than appending it everywhere - what would you think about sticking it up top next to `middleware.Tenancy`? It is doing something really similar and we'll want to do it for basically every request that we have tenancy on right now as well.

Reply:
yes it is much better - done  I change second commit - https://github.com/RedHatInsights/sources-api-go/pull/420/commits/dea0f1c73ac3b5ba36383c84eb160b38fd8b1a78 

thanks!